htdocs/web_portal/GOCDB_monitor/check.php

```diff
@@ -1,6 +1,9 @@
 <?php
 require_once "tests.php";
 
+// Initialise local_info.xml configuration overrides
+\Factory::getConfigService()->setLocalInfoOverride($_SERVER['SERVER_NAME']);
```

Comment:
The backslash is about namespacing, right? Is it needed here?

Reply:
Yes it's namespace spec and no, it's not needed here I think. However, as it is the most common usage in GOCDB codebase, I'll keep as is but make other references in this module follow the same.